examples/text_classification/tc_mnli_xlnet.ipynb

```diff
@@ -0,0 +1,970 @@
+{
```

Comment:
<p>mmm shouldn't these sum to one per column in your case? I have normally seen the confusion matrix transposed, row true and cols pred, have you seen more of the other way?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/nlp/pull/386/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>

Reply:
1. good catch. it should be rows true and cols pred - will fix that.
2. confusion matrix usually use the absolute number for each cell, we decided to use % due to the large numbers. will review if it makes more sense to sum every column to one.